hnix.cabal

```diff
@@ -516,7 +872,7 @@ library
     , monadlist >= 0.0.2 && < 0.1
     , mtl >= 2.2.2 && < 2.3
     , optparse-applicative >= 0.14.3 && < 0.15 || >= 0.15.0.0 && < 0.16
-    , parser-combinators >= 1.0.1 && < 1.1 || >= 1.2.0 && < 1.3
+    , parser-combinators
```

Comment:
Bump the upper bound here instead of dropping them entirely?

Reply:
Yeah, better idea that.